dist/component/component-war/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -27,5 +27,7 @@
       <module name="org.jboss.resteasy.resteasy-jackson2-provider" services="import"/>
       <module name="org.hawkular.bus"/>
     </dependencies>
+    <module name="org.infinispan" export="true" />
+    <module name="org.infinispan.commons" export="true"/>
```

Comment:
This is not going to have any effect.  In fact, I thought I had removed the WAR-level jboss-deployment-structure.xml.  The WAR-level instances of this file are ignored, only the EAR-level version is relevant.